search/grammar_v2_index.go

```diff
@@ -172,7 +167,7 @@ func IndexGrammars(esc *elastic.Client, indexDate string, grammars GrammarsV2, v
 						HitType:      grammar.HitType,
 						Intent:       intent,
 						Rules:        rules,
-						RulesSuggest: SuggestField{es.Unique(assignedRulesSuggest), float64(100)},
+						RulesSuggest: es.SuggestField{es.Unique(assignedRulesSuggest), float64(100)},
```

Comment:
Why float64(100)? The default you use in index is 50.
Please use constant...

Reply:
TBD constant